Casks/t/taobao.rb

```diff
@@ -0,0 +1,31 @@
+cask "taobao" do
+  arch arm: "arm64", intel: "x64"
+
+  version "2.1.1"
+  sha256 arm:   "a92c56fe614ad7fe0bacfceda8dde6d2dad11bcb0c4e13cd30728fee3fe16683",
+         intel: "cf8839156c8f23e37656e775a2f6630f170e43440c79cea575ab2fb1f15a1cec"
+
+  url "https://tblifecdn.taobao.com/taobaopc/shop/taobao-setup-darwin-#{arch}-#{version}.dmg"
+  name "taobao"
+  name "淘宝桌面版"
+  desc "Shopping client"
+  homepage "https://pc.taobao.com/"
+
+  livecheck do
+    url "https://tblifecdn.taobao.com/taobaopc/updater/darwin/arm64/latest-mac.yml"
```

Comment:
```suggestion
    url "https://tblifecdn.taobao.com/taobaopc/updater/darwin/#{arch}/latest-mac.yml"
```

Sorry @RyanHuV I missed this before.